deployments/puppet/README.md

```diff
@@ -90,6 +90,8 @@ configuration changes in order for auto instrumentation to take effect.
 |:------------------------------------------------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------|
 | `with_auto_instrumentation`                                 | **Linux only**: Whether to install/manage [Splunk OpenTelemetry Auto Instrumentation for Java](https://github.com/signalfx/splunk-otel-collector/tree/main/instrumentation). When set to `true`, the `splunk-otel-auto-instrumentation` deb/rpm package will be downloaded and installed from the Collector repository.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               | `false`                                                     |
```

Comment:
I guess that this one should be removed, since, now it is following the pattern of a list of instrumentations to be deployed.

Reply:
I can fix the description to match chef: https://github.com/signalfx/splunk-otel-collector/blob/main/deployments/chef/README.md?plain=1#L189

I don't think I can deprecate it quite yet.

Reply:
I don't understand why it can't be moved to the new one. I mean the functionality is the same, but, the argument name is different.

Reply:
just out of caution, if you look at the parameters we take in general, we don't remove them away

Reply:
Eventually we will :)